src/components/ResourceBrowser/ResourceList/ResourceList.tsx

```diff
@@ -315,7 +319,6 @@ export default function ResourceList() {
             !isNodes
         ) {
             selectedResource.gvk.Kind !== SIDEBAR_KEYS.nodeGVK.Kind && getResourceListData()
-            setSearchText('')
```

Comment:
Need to remove this cuz on a page refresh `SelectedResource` changes from `null -> Node -> (finally correct type)`